app/models/ai_persona.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+class AiPersona < ActiveRecord::Base
+  class MultisiteHash
+    def initialize(id)
+      @hash = Hash.new({})
```

Comment:
I think you want `Hash.new { |h, k| h[k] = {} }`, otherwise the same default hash aliases for every key.

Reply:
good catch, thanks fixing